runtime/observability/telemetry-provider-otel/jvm/src/aws/smithy/kotlin/runtime/telemetry/otel/OtelMeterProvider.kt

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package aws.smithy.kotlin.runtime.telemetry.otel
+
+import aws.smithy.kotlin.runtime.telemetry.context.Context
+import aws.smithy.kotlin.runtime.telemetry.metrics.*
+import aws.smithy.kotlin.runtime.util.Attributes
+import io.opentelemetry.api.OpenTelemetry
+import io.opentelemetry.api.metrics.ObservableDoubleMeasurement
+import io.opentelemetry.api.metrics.ObservableLongMeasurement
+import io.opentelemetry.api.metrics.DoubleHistogram as OtelDoubleHistogram
+import io.opentelemetry.api.metrics.LongCounter as OtelLongCounter
+import io.opentelemetry.api.metrics.LongHistogram as OtelLongHistogram
+import io.opentelemetry.api.metrics.LongUpDownCounter as OtelLongUpDownCounter
+import io.opentelemetry.api.metrics.Meter as OpenTelemetryMeter
+
+internal class OtelMeterProvider(private val otel: OpenTelemetry) : MeterProvider {
+    override fun getOrCreateMeter(scope: String, attributes: Attributes): Meter {
+        val meter = otel.getMeter(scope)
+        // FIXME: meter level attributes not supported yet by Java impl
+        // https://github.com/open-telemetry/opentelemetry-java/issues/5503
+        return OtelMeter(meter)
+    }
```

Comment:
**Question**: Should we warn or throw an exception if non-empty attributes are passed?

Reply:
Part of me is thinking we just remove `attributes` from the `getOrCreate` level interfaces (`MeterProvider` and `TracerProvider`).